src/main/java/poolsheen/Ui.java

```diff
@@ -0,0 +1,143 @@
+package poolsheen;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import poolsheen.task.Task;
+
+/**
+ * Represents everything that involves interactions with the user.
+ */
+public class Ui {
+    /** The indentation for all printed Poolsheen lines. */
+    public static final String BEGIN_SPACE = "      ";
+
+    /** The Poolsheen logo. */
+    private static final String LOGO = "\n" + "POOLSHEEN";
+
+    /** The message first printed when the Poolsheen program runs. */
+    private static final String WELCOME_MESSAGE = "Hello from" + LOGO + "\n" +
+            "Type something in for Poolsheen to respond to you:";
+
+    /** The message printed last whenever Poolsheen replys. */
+    private static final String LAST_REPLY = "meow *_*";
+
+    /** A horiontal line to distinguish each interaction the user has with Poolsheen. */
+    private static final String HORIZONTAL_LINE = "---------------";
+
+    /** The message last printed when the Poolsheen program ends. */
+    private static final String GOODBYE_MESSAGE = "MeoAww... See you next time :(\n"
```

Comment:
Great idea to abstract all string constants here! Makes the job easier should you want to change the UI messages at any point in the future. I also noted that you've capitalised the variable names; it's a common mistake that you've avoided. Good job!